### DISCUSSION TITLE
ESPN_S2 and SWID

### DISCUSSION THREAD

**Discussion by TPassalacqua** [cwendt94/espn-api#475] [ESPN API] [answered]
Hi, I can't find where i get these tokens. I've looked for a developer portal on ESPN but it seems they do not have one? Or maybe I'm looking in the wrong place. Any direction would be much appreciated!!

thanks!

**Comment by dtcarls** [comment, marked as answer]
https://github.com/cwendt94/espn-api/discussions/150

**Reply by TPassalacqua** [reply]
Thank you! I didn't see this when I looked!
